pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -568,6 +568,27 @@
             "#import": [ "opensuse.42.1", "opensuse-x64" ]
         },
 
+        "gentoo": {
+            "#import": [ "linux" ]
+        },
+        "gentoo-x64": {
+            "#import": [ "gentoo", "linux-x64" ]
+        },
+
+        "buildroot": {
+            "#import": [ "linux" ]
+        },
+        "buildroot-x64": {
+            "#import": [ "buildroot", "linux-x64" ]
+        },
+
+        "buildroot.2017.02.2": {
+            "#import": [ "buildroot" ]
+        },
+        "buildroot.2017.02.2-x64": {
```

Comment:
(I'm a complete buildroot newbie, so take this with a grain of salt)

It looks like there are "real" releases every 3 months for buildroot (not including daily builds). https://buildroot.org/downloads/

Including all of (current and future) versions does seem annoying though. It would be nice if the "buildroot-x64" RID was assumed if you were running on some other buildroot-date-x64 system that wasn't in the graph.